operations/alloy-mixin/dashboards/utils/templates.libsonnet

```diff
@@ -0,0 +1,100 @@
+local dashboard = import './dashboard.jsonnet';
+
+{
+    newTemplateVariablesList(filterSelector='', enableK8sCluster=true, includeInstance=true, useSentenceCaseLabel=false):: {        
```

Comment:
This doesn't need to return an object; if you return the array directly you can  avoid needing to add `.variables` every time you use the function.

Here's an example:

```jsonnet 
{
  myFunction():: (
     local variable_1 = 15; 
     local variable_2 = 30; 

     variasble_1 + variable_2 
  ),
}
```

The surrounding parenthesis are optional, but helps with readability. The semicolon after each local variable is required, and the final expression must not end in a semicolon. 

The surrounding parenthesis are optional, but helps with readability. If you omit them it looks like this:

```jsonnet 
{
  myFunction():: 
     local variable_1 = 15; 
     local variable_2 = 30; 

     variasble_1 + variable_2,
}
```